frontend/src/routes/(authenticated)/+page.ts

```diff
@@ -22,10 +22,31 @@ export async function load(event: PageLoadEvent) {
         }
   `), {});
 
+  const myDraftProjectPromise = await client.awaitedQueryStore(event.fetch, graphql(`
```

Comment:
It's actually possible to query all the data you want in one request instead of making 2. Eg
```gql
query loadProjects {
    myProjects {
    ...
    }

    myDraftProjects {
    ...
    }
}
```

Reply:
Just made the change. Thank you!
[#712ac6f](https://github.com/sillsdev/languageforge-lexbox/pull/872/commits/712ac6fdc15c4ba452e2e68457ec0d70f5fe4af2)